luigi/notifications.py

```diff
@@ -358,7 +358,8 @@ def _prefix(subject):
     If the config has a special prefix for emails then this function adds
     this prefix.
     """
-    if email().prefix is not None:
+    prefix = email().prefix
+    if prefix is not None and prefix != '':
```

Comment:
Thanks. Though these two lines can be simplified to simply:

`if email().prefix:`

This is awesome. As we also get rid of the `is not None` antipattern in python!